### PR TITLE
removes the "and clear buffer" text on eject button

### LINKED ISSUE #34247: s/Eject and Clear Buffer/Eject/ on the ChemMaster 3000
For some reason this button has had the "and Clear Buffer" addition ever since that feature was removed a long time ago.

### PR DESCRIPTION
**Pull request by yorii**
:cl:
spellcheck: Renamed the ChemMaster eject button to highlight the fact that the buffer does not get cleared
/:cl:

[why]: It doesn't clear the buffer since a while back.

Fixes #34247

### CODE REVIEW

**Comment by Cyberboss** [comment]
Hi, thanks for contributing to /tg/station.

While you did the correct thing updating the template, you also need to recompile TGUI for the change to take effect. You can do this by installing node.js (make sure it's in your PATH), running /tgui/install_dependencies.bat, and then running /tgui/build_assets.bat.

Alternatively, if that's too involved for you, you can use our community provided online compiler [here](https://dextraspace.net/TGUICompiler). To use it, upload chem_master.ract and hit "Submit". You'll be provided a .zip file containing the updated tgui.css and tgui.js, which you should then use to replace the existing ones in /tgui/assets/.

All current TGUI documentation can be browsed [here](https://github.com/tgstation/tgstation/blob/master/README.md).

**Comment by Cyberboss** [comment]
Also your changelog is malformed, here is how it should be written
```
:cl:
spellcheck: Renamed the ChemMaster eject button to highlight the fact that the buffer does not get cleared
/:cl:
```

**Review by Cyberboss — CHANGES_REQUESTED** [review]
See above

**Comment by Cyberboss** [comment]
Also add `Fixes #34248` to your PR body to automatically close the issue you opened when this gets merged.

**Comment by yorii** [comment]
Thank senpai

**Comment by Robustin** [comment]
>First PR
>Needs to recompile TGUI

Oof

File: tgui.css [hunk 1]
```diff
@@ -0,0 +1 @@
...ht:4px}body.syndicate div.subdisplay{width:100%;margin:0}body.syndicate header.titlebar .close,body.syndicate header.titlebar .minimize{display:inline-block;position:relative;padding:7px;margin:-7px;color:#e74242}body.syndicate header.titlebar .close:hover,body.syndicate header.titlebar .minimize:hover{color:#eb5e5e}body.syndicate header.titlebar{position:fixed;z-index:1;top:0;left:0;width:100%;height:32px;background-color:#363636;border-bottom:1px solid #161616;box-shadow:0 3px 3px rgba(0,0,0,.1)}body.syndicate header.titlebar .statusicon{position:absolute;top:4px;left:12px;transition:color .5s}body.syndicate header.titlebar .title{position:absolute;top:6px;left:46px;color:#e74242;font-size:16px;white-space:nowrap}body.syndicate header.titlebar .minimize{position:absolute;top:6px;right:46px}body.syndicate header.titlebar .close{position:absolute;top:4px;right:12px}.no-icons header.titlebar .statusicon{font-size:20px}.no-icons header.titlebar .statusicon:after{content:"O"}.no-icons header.titlebar .minimize{top:-2px;font-size:20px}.no-icons header.titlebar .minimize:after{content:"—"}.no-icons header.titlebar .close{font-size:20px}.no-icons header.titlebar .close:after{content:"X"}
```
Comment (Cyberboss) [inline tgui.css]: This should be in /tgui/assets/

File: tgui.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...",{imul:function(t,e){var n=65535,a=+t,r=+e,i=n&a,o=n&r;return 0|i*o+((n&a>>>16)*o+i*(n&r>>>16)<<16>>>0)}})},{22:22,24:24}],107:[function(t,e,n){var a=t(22);a(a.S,"Math",{log10:function(t){return Math.log(t)/Math.LN10}})},{22:22}],108:[function(t,e,n){var a=t(22);a(a.S,"Math",{log1p:t(50)})},{22:22,50:50}],109:[function(t,e,n){var a=t(22);a(a.S,"Math",{log2:function(t){return Math.log(t)/Math.LN2}})},{22:22}],110:[function(t,e,n){var a=t(22);a(a.S,"Math",{sign:t(51)})},{22:22,51:51}],111:[function(t,e,n){var a=t(22),r=t(49),i=Math.exp;a(a.S+a.F*t(24)(function(){return-2e-17!=!Math.sinh(-2e-17)}),"Math",{sinh:function(t){return Math.abs(t=+t)<1?(r(t)-r(-t))/2:(i(t-1)-i(-t-1))*(Math.E/2)}})},{22:22,24:24,49:49}],112:[function(t,e,n){var a=t(22),r=t(49),i=Math.exp;a(a.S,"Math",{tanh:function(t){var e=r(t=+t),n=r(-t);return e==1/0?1:n==1/0?-1:(e-n)/(i(t)+i(-t))}})},{22:22,49:49}],113:[function(t,e,n){var a=t(22);a(a.S,"Math",{trunc:function(t){return(t>0?Math.floor:Math.ceil)(t)}})},{22:22}],114:[function(t,e,n){"use strict";var a=t(46),r=t(29),i=t(30),o=t(11),s=t(81),p=t(24),u=t(74).trim,c="Number",l=r[c],d=l,f=l.prototype,h=o(a.create(f))==c,m="trim"in String.prototype,g=function(t){
```
Comment (Cyberboss) [inline tgui.js]: Ditto

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: tgui.js (hunk 2 above)
Comment (yorii) [reply tgui.js]: you mean they're not? shit I fucked up

**Comment by yorii** [comment]
Well I managed the compilation just fine, the whole ordeal with trying to figure out how to revert a commit or even delete a file is giving me a headache though. The webui is insanely crippled in that regard..

**Comment by yorii** [comment]
I think I got it right now, but I made a huge mess of commits :)

**Review by Cyberboss — APPROVED** [review]

**Comment by nicbn** [comment]
Don't use webeditor, download git and do it the right way next time

**Pull request merged**